File: src/main/java/template/Part1.java
package template;

import util.Utility;

import java.util.List;

public class Part1 {

    public static void main(String[] args) {
        var testInput = Utility.readLines("dayxx/test_1");
        Utility.check(0, solve(testInput));

        var input = Utility.readLines("dayxx/input");
        System.out.println(solve(input));
    }

    private static int solve(List<String> input) {
        return 0;
    }
}

File: src/main/kotlin/template/Part1.kt
package template

import println
import readInput

fun main() {
    fun solveLine(line: String): Int {
        return 0
    }

    fun solve(lines: List<String>): Int {
        return lines.sumOf { l -> solveLine(l) }
    }

    // test
    val testInput = readInput("day0x/test_1")
    check(solve(testInput) == 0)

    val input = readInput("day0x/input")
    solve(input).println()
}
